server/core/region.go

```diff
@@ -39,7 +40,8 @@ func errRegionIsStale(region *metapb.Region, origin *metapb.Region) error {
 }
 
 // RegionInfo records detail region info.
-// Read-Only once created.
+// most properties is Read-Only once created exclude buckets.
+// buckets should be modified through report buckets and the version is greater than the current.
```

Comment:
```suggestion
// the `buckets` could be modified by the request `report buckets` with greater version.
```
Does the above make sense?